docs/modules/servers/partials/configure/blobstore.adoc

```diff
@@ -126,6 +126,29 @@ BucketPrefix is the prefix of bucket names in James BlobStore
 Unless a special case like storing blobs of deleted messages.
 |===
 
+==== SSE-C Configuration
+
+.SSE-C configuration
+|===
+| Property name | explanation
+
+| encryption.s3.sse.c.enable
+| optional: Boolean. Default is false. Controls whether to use Server-Side Encryption with Customer-Provided Keys (SSE-C) for S3 blobs.
+
+| encryption.s3.sse.c.master.key.algorithm
+| String. Required if `encryption.s3.sse.c.enable` is true. The algorithm used to derive the master key from the provided password. Eg: AES256
+
+| encryption.s3.sse.c.key.derivation.enabled
+| optional: Boolean. Default is false. Controls whether to derive the encryption key from the bucket name as master key and blob-id as salt.
+
+| encryption.s3.sse.c.master.key.password
+| String. Required if `encryption.s3.sse.c.key.derivation.enabled` is false. The password used to derive the customer key.
```

Comment:
This makes me confused. 
`Required if encryption.s3.sse.c.key.derivation.enabled is false` => `encryption.s3.sse.c.master.key.password` is needed when key derivation is **disabled**.

Then reading the remaining part `The password used to derive the customer key` => why it is used to derive when derivation was disabled before?

I think it was 2 different "derivations", no? Then we should rephrase a bit the `The password used to derive the customer key.` part IMO.

Reply:
+1